worlds/alttp/docs/oof_sound_fr.md

```diff
@@ -0,0 +1,21 @@
+# Guide de customisation du son "OOF"
+
+## À quoi sert cette fonctionnalité ?
+
+Cela remplace l'effet sonore quand Link prend des dégâts. Le cas d'utilisation prévu pour cela est celui des sprites personnalisés, mais vous pouvez utiliser avec n'importe quel sprite, celui par défaut inclus. 

```

Comment:
```suggestion
Elle permet de modifier l'effet sonore quand Link prend des dégâts. Elle est habituellement utilisé en paire avec l'usage d'un sprite personnalisé. Cependant, il est possible d'utiliser cette fonctionnalité avec le sprite par défaut.
```
The formulation is strange, it felt translated from french using tool. Rewrote it in a more natural sounding way